packages/oidc-middleware/src/connectUtil.js

```diff
@@ -35,6 +35,11 @@ connectUtil.createOIDCRouter = context => {
   } = context.options.routes;
   oidcRouter.use(loginPath, bodyParser.urlencoded({ extended: false}), connectUtil.createLoginHandler(context));
   oidcRouter.use(callbackPath, connectUtil.createCallbackHandler(context));
+  oidcRouter.use(function(err, res, res, next){
```

Comment:
just for consistency, we should be using the `() => {}` syntax for anonymous functions. Sounds like something I should add to our linter